packages/v4/src/content/design-guidelines/extensions/log-viewer/log-viewer.md

```diff
@@ -0,0 +1,49 @@
+---
+id: Log viewer
+section: extensions
+---
+
+A log viewer is a preconfigured component that gives you the option to visualize your log content. Log viewer renders log output in real time in a clear and structured way.
+
+## Light theme log viewer
+
+<img src="./img/logviewer.png" alt="Log Viewer"/>
+
+Type of log dropdown menu: Allow users to switch between different types of logs.
```

Comment:
This should be the first bullet! you forgot to number this and everything is off haha